models/tridimensional/docking_validation/generate_pams_chimera.py

```diff
@@ -29,13 +27,15 @@ def mutate_nt(pam_idx, base):
     runCommand("swapna " + complement_base + " : " + complement_pos)
 
 
-def generate_pdb(template_pdb_basename, pam_seq):
+def generate_pdb(pam, input_pdb, output_dir):
```

Comment:
I don't think this should be called generate_pdb -- need something that says "pam variant" and "chimera"